app/src/renderer/scripts/utils.js

```diff
@@ -0,0 +1,14 @@
+import createHash from "create-hash"
+import varint from "varint"
+import b64 from "base64-js"
+
+export default function(txstring) {
```

Comment:
The script is called utils and exports a function that converts tx strings into json? I think this is not really understandable.
How about a script called tx-utils.js that exports a function named parseTx?

Reply:
I agree it should be named, could be something like `getTxHash`

Reply:
makes sense, i think `getTxHash` is an accurate name